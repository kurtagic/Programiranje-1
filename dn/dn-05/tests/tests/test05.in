32
3 5
7 5
7 6
11 7
1 7
1 7
3 5
14 7
2 6
1 6
1 7
14 6
8 7
14 7
3 7
3 5
8 5
1 5
8 6
7 6
1 5
8 7
11 5
14 7
3 7
14 5
8 6
2 5
1 6
14 7
0 6
11 5
