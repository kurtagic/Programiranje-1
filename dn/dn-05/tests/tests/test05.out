5: 52
6: 54
7: 51
8: 51
9: 51
